src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveforbarn/BarnFyllerÅrOppfølgingsoppgaveService.kt

```diff
@@ -84,7 +124,7 @@ class BarnFyllerÅrOppfølgingsoppgaveService(
 
     private fun opprettOppgaveForBarn(barn: List<BarnTilUtplukkForOppgave>): List<OpprettOppgaveForBarn> =
         barn.mapNotNull {
-            Alder.fromFødselsdato(fødselsdato(it))?.let { alder ->
+            Alder.fromFødselsdato(hentFødselsdatoFraGrunnlagsdata(it))?.let { alder ->
```

Comment:
Dette vil ikke fungere for terminbarn, siden de, mest trolig, ikke ligger i grunnlagsdataene. Her må vi kanskje bruke fødselsdatoen vi hentet ut tidligere, men som vi ikke tok med videre i `BarnTilUtplukkForOppgave`